app/controllers/concerns/hydranorth/batch_controller_behavior.rb

```diff
@@ -3,21 +3,19 @@ module BatchControllerBehavior
     extend ActiveSupport::Concern
     include Sufia::BatchControllerBehavior
     include Hydra::Catalog
-    include Hydranorth::Collections::SelectsCollections
+    include Hydranorth::Collections::CollectionSelection
+    include Hydranorth::Collections::CommunitySelection
 
     included do
-      before_action only: [:edit] do
-        find_communities_with_edit_access
-        find_collections_with_edit_access
-      end
-
       class_attribute :edit_form_class, :cstr_edit_form_class, :ser_edit_form_class
       self.edit_form_class = Hydranorth::Forms::BatchEditForm
       self.cstr_edit_form_class = Hydranorth::Forms::CstrBatchEditForm
       self.ser_edit_form_class = Hydranorth::Forms::SerBatchEditForm
     end
 
     def edit
+      @user_communities = find_communities(access_levels[:edit])
+      @user_collections = find_collections(access_levels[:edit])
```

Comment:
Whoops, yeah.

The goal is to get rid of anywhere "hidden" that magically assigns to variables like @user_collection that get used in the view, and make it all explicit in the controller so we can see where this stuff is coming from (and prevent bugs where we munge it for one use-case and then it gets silently shared with some other controller/view accidentally).